packages/compiler/test/cli.test.ts

```diff
@@ -34,6 +34,15 @@ describe("compiler: cli", () => {
       });
     });
 
+    it("--config plumbs through", async () => {
+      const options = await resolveCompilerOptions({ config: `${cwd}/myConfigDir` });
```

Comment:
looks like it is trying to load config here, perhaps we need to create a directory and place a default config file?

Reply:
Well, this test was created when I didn't think I had a better way to test the config loading. With the new tests added, I feel this is unnecessary so I removed it.

Reply:
Even pointing it to an existing config the test code would fail even though it runs fine locally.